order.go

```diff
@@ -312,9 +312,11 @@ type Order struct {
 	CurrentSubtotalPrice     *decimal.Decimal        `json:"current_subtotal_price,omitempty"`
 	TotalDiscounts           *decimal.Decimal        `json:"total_discounts,omitempty"`
 	TotalDiscountSet         *AmountSet              `json:"total_discount_set,omitempty"`
```

Comment:
Let's remove this guy - if it's broken then a backward incompatible change will be a good heads up for people that used the field that this was never working.

Reply:
Yep makes sense, I've removed that.